create_env.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+pyv=$(python3 -c 'import sys; print(sys.version_info[1])')
+echo $pyv
+
+if [[ "$pyv" -eq "9" ]]
+then 
+    echo "Valid version"
+    python3 -m venv --clear venv_quant_course_v39
+    source ./venv_quant_course_v39/bin/activate #lehet a Scriptben van bizonyos distrokon
```

Comment:
If that's the case, can we check if the file exists and try the other location otherwise?

Reply:
Yes I implemented and tested these checks